### DISCUSSION TITLE
How make a contribuiton to translations in github/docs

### DISCUSSION THREAD

**Discussion by ramonpaolo** [community/community#23441] [New to GitHub]
<p>How can I help with github/docs translations?</p>
<p>I recently opened a PR, however, the documentation for translates cannot be changed : (</p>

**Comment by jsoref** [comment, marked as answer]
<p>Afaiu, you can’t.</p>
<p>The translations are managed at <a href="http://crowdin.com">crowdin.com</a>, but I don’t think that it’s an open translation thing.</p>
<p>You should be able to file a bug against the translation and explain why it’s bad and rely on the docs team to send along the message.</p>
